src/__tests__/app.test.js

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from "@testing-library/react";
+import App from "../components/App";
+
+describe("App", () => {
+  it("renders App correctly", () => {
+    const { asFragment } = render(<App />);
+    const nasaImg = screen.getByAltText("NASA in red stylised font");
+
+    expect(asFragment()).toMatchSnapshot();
+    expect(nasaImg).toHaveAttribute("src");
```

Comment:
Did you know you can search not just for the attributes on an element, but also its value? EG:
```
expect(nasaImg).toHaveAttribute("src", 'https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg');
```
Makes for a more robust test if you can verify its exactly what you expect